frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -111,23 +111,23 @@ export default Vue.extend({
     props: {
         liquidationLimits: {
             type: Object as Vue.PropType<LiquidationLimits>,
+            default: undefined,
```

Comment:
Same here, please set back properties to be required. Here and for all other props in this file